src/Components/Selectors.js

```diff
@@ -0,0 +1,12 @@
+import { createSelector } from 'reselect';
+
+// Input selector (returns the array of rockets)
+const selectRockets = (state) => state.rockets.rockets;
+
+// Memoized selector that derives the reserved rockets
+const selectReservedRockets = createSelector(
+  [selectRockets],
+  (rockets) => rockets.filter((rocket) => rocket.reserved === true),
+);
+
+export default selectReservedRockets;
```

Comment:
I hereby refer to our technical we do not need to preserve user data:

<img width="406" alt="image" src="https://github.com/George7h/space-travelers/assets/101390796/a448d4bb-37b6-4274-b5e5-20ee57b2674e">